### DISCUSSION TITLE
blog/the-ins-and-outs-of-hash-tables/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#1] [Q&A]
# blog/the-ins-and-outs-of-hash-tables/

Hash Tables are one of the most used Data structures, they’re so popular that almost every language has their own implementation and nomenclature of Hash Tables.
One very interesting fact about Hash Tables is that they’re also used as building blocks for:
Classes and its members Variable lookup table As we can see, Hash Tables are not just used in the business case of any application, but it’s also used in the inner working of any programming language.

http://localhost:1313/blog/the-ins-and-outs-of-hash-tables/

**Comment by Davshiv20** [comment]
Great Blog! Interesting read

**Reply by Jitesh117** [reply]
Thanks! Really glad you enjoyed it :)
